scripts/delete_kibana_indices.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Set the default config file
+DEFAULT_CONFIG_FILE="./config/kibana.dev.yml"
+
+# Check if a config file is provided and use the default if not
+if [ "$#" -eq 1 ]; then
+    CONFIG_FILE="$1"
+else
+    CONFIG_FILE="$DEFAULT_CONFIG_FILE"
+fi
+
+# Parse the YAML file using grep and sed to extract the required values
+ELASTICSEARCH_HOSTS=$(grep -E '^elasticsearch.hosts: ' "$CONFIG_FILE" | sed 's/.*: //')
+ELASTICSEARCH_USERNAME=$(grep -E '^elasticsearch.username: ' "$CONFIG_FILE" | sed 's/.*: //')
+ELASTICSEARCH_PASSWORD=$(grep -E '^elasticsearch.password: ' "$CONFIG_FILE" | sed 's/.*: //')
+
+# Set default values if the extracted values are empty
+[ -z "$ELASTICSEARCH_HOSTS" ] && ELASTICSEARCH_HOSTS="http://localhost:9200"
+[ -z "$ELASTICSEARCH_USERNAME" ] && ELASTICSEARCH_USERNAME="system_indices_superuser"
+[ -z "$ELASTICSEARCH_PASSWORD" ] && ELASTICSEARCH_PASSWORD="changeme"
+
+# Get the list of indices from the _cat/indices API
+echo "Getting list of indices..."
+INDICES=$(curl -s -w "%{http_code}" -X GET "${ELASTICSEARCH_HOSTS}/_cat/indices/.kibana*?format=txt" \
+     -u "${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}" | awk '{print $3}')
+HTTP_STATUS="${INDICES##* }"
+INDICES="${INDICES% *}"
+if [ $? -ne 0 ] && [ "$HTTP_STATUS" != "404" ]; then
```

Comment:
exit here if it was a 404?

Reply:
Let me look into this, I was using another API at first, not sure if _cat/indices returns a 404 if no indices match the pattern. If it does, I still have to trigger an update to the config file to restart the server. If it doesn't, a 404 should be a failure.

Reply:
Simplified it a bit, no need to special-case a 404 here. Thanks!